.github/ISSUE_TEMPLATE/1_bug_report.yaml

```diff
@@ -1,43 +1,18 @@
 name: Report a bug
 description: Any errors that are not UI related.
-labels: ["needs triage", "bug"]
+labels: ["needs triage"]
 body:
-  - type: markdown
```

Comment:
There's still value in instructing users to 
1. See if issue already exist
2. Read the docs
3. Ask questions in slack

Reply:
Sure, but the page where you choose what kind of issue you have already has links to our slack and docs - so if we have it here too then you will see it twice before being able to report your bug